src/components/application_manager/include/application_manager/commands/request_from_hmi.h

```diff
@@ -78,6 +80,12 @@ class RequestFromHMI : public CommandImpl, public event_engine::EventObserver {
                          const hmi_apis::Common_Result::eType result_code,
                          const std::string error_message);
 
+  /**
+   * @brief Return HMI application id.
+   * @return hmi_app_id as uint32_t.
+   */
+  uint32_t HMIAppId();
```

Comment:
@IGapchuk if it is getter, use appropriate coding style. Also make it const

Reply:
@IGapchuk you added this getter in base class. Looks like it widely used functionality. Please replace all places where this getter could be used.

Reply:
@LuxoftAKutsan All implamentation has changed in da8a1b010377dda6ee7a1683c3439e530339ea0b.